intellij-ide.md

```diff
@@ -132,13 +121,23 @@ your changes near instantly.
 
 To edit your code and hot reload changes:
 
-1. Run your app in debug mode.
+1. Start your app with 'Run' (without breakpoints) or 'Debug' (with breakpoints).
 
-1. Make desired changes in your source code (without stopping the debugger).
+1. Make desired changes in your source code
 
-1. In the Debug panel at the bottom, click the **Hot Reload button**.
+1. Do not press the 'Stop' button; rather, to see your changes just click the
+**Hot Reload button**.
 
-![Main IntelliJ toolbar](/images/intellij/debug-toolbar.png)
+### Advanced debugging with Observatory
+
+Observatory is an additional debugging and profiling tool presented with an
+html-based UI. For details see the [Observatory page](https://dart-lang.github.io/observatory/).
+
+To open Observatory:
+
+1. Run your app in debug mode.
+
+1. In the Debug panel at the bottom, click the **Arrow icon** ('Open Observatory').
```

Comment:
It's since transmuted into a stopwatch icon:

<img width="100" alt="screen shot 2017-01-23 at 8 52 13 am" src="https://cloud.githubusercontent.com/assets/1269969/22213490/4d582f90-e149-11e6-8b6a-8acc86ffcd34.png">